packages/components-angular/tests/vrt/specs/styles/flyout-grid.vrt.ts

```diff
@@ -1,9 +1,15 @@
 import { expect, test } from '@playwright/test';
+import {
+  viewportWidth3XL,
+  viewportWidth4XL,
+  viewportWidths,
+  viewportWidthXXL,
+} from '@porsche-design-system/shared/testing/playwright.vrt.config';
 
 const style = 'styles-flyout-grid';
 
 test.describe(style, async () => {
-  [320, 480, 760, 1000, 1300, 1760, 1920, 2560, 3000].forEach((viewportWidth) => {
+  ([...viewportWidths, viewportWidthXXL, viewportWidth3XL, viewportWidth4XL] as const).forEach((viewportWidth) => {
```

Comment:
Maybe we could use something like `furtherExtendedViewports ` again?

Reply:
Also thought about it, but it's not really always the exact same extended viewports that we need.